MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/dds/CreateHotspotsActivity.java

```diff
@@ -148,7 +155,7 @@ private void addClusteredGeoJsonSource(MapboxMap mapboxMap) {
             Expression.lt(pointCount, literal(layers[i - 1][0]))
           )
       );
-      mapboxMap.addLayerBelow(circles, "building");
+      mapboxMap.getStyle().addLayerBelow(circles, "building");
```

Comment:
maybe change addClusterGeoJsonSource signature to take Style as parameter? This makes it more clear to readers that a style is required to call this method.

Reply:
fixed